boxsdk/session/session.py

```diff
@@ -289,6 +290,20 @@ def _raise_on_unsuccessful_request(network_response: 'NetworkResponse', request:
                 network_response=network_response,
             )
 
+    @staticmethod
+    def _validate_json_response(network_response: 'NetworkResponse', request: '_BoxRequest') -> bool:
+        """
+        Validate that the response is json if the request expects json response.
+
+        :param network_response:
+            The network response which is being tested for success.
+        :param request:
+            The API request that could be unsuccessful.
+        """
+        if request.expect_json_response and not is_json_response(network_response):
```

Comment:
could be simplified to simple return statement.

Reply:
if function return bool it is a good practice to name functions strating from is_...

Reply:
maybe _is_json_response_if_expected, or sth like this